src/Mealz/MealBundle/Controller/MealController.php

```diff
@@ -139,12 +139,13 @@ private function checkProfile(?string $profileId): ?Profile
     private function generateResponse(string $route, string $action, Meal $meal, Participant $participant): JsonResponse
     {
         $bookedDishes = [];
-        $partCombinedDishes = $participant->getCombinedDishes();
-        if (0 < $partCombinedDishes->count()) {
-            $bookedDishes = $partCombinedDishes->map(fn (Dish $dish) => $dish->getSlug())->toArray();
+        $dishes = $participant->getCombinedDishes();
+        if (0 < $dishes->count()) {
+            $bookedDishes = array_map(fn (Dish $dish) => $dish->getSlug(), $dishes->toArray());
         }
 
         return new JsonResponse([
+            'id' => $participant->getId(),
```

Comment:
Take care! I think this has to match with the typescript interface. If you expect a response without ParticipantID (SwapResponse) the callback isn't called. Please, check where generateResponse is used and which actions are called.

Reply:
Interfaces are already updated for the change.

Swap response actually contains participant ID.

Reply:
Yeah, I meant "with ParticipantID". :-D Alright